packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Writing a unit test
+tags: Mocha, Chai, Enzyme, schemaform-utils,
+---
+
+# Writing a unit test
+
+## Unit testing overview
+
+- `vets-website` uses
+  - [Mocha](https://mochajs.org/) for running unit tests
+  - [Chai](http://chaijs.com/) for test assertions
+  - [Enzyme](https://airbnb.io/enzyme/) for mounting and inspecting React components
+- Unit tests are **collocated with application folder** in a test directory that matches the application directory structure
+```
+- 📂 MyApplication
+  - 📂 components
+    - MyComponent.jsx
+  - 📂 tests
+    - 📂 components
+      - MyComponent.unit.spec.jsx
+```
+- Mocha runs all unit tests under `/src` folder that end with `.unit.spec.js(x)`
+
+## Unit test conventions
+```
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import MyComponent from '../../components/MyComponent';
+
+describe('MyApplication', () => {
+  describe('MyComponent', () => {
+    it('renders privacy act disclosure when state.show is true', () => {
+        const componentUnderTest = shallow(<MyComponent />);
+        ...
+        expect(someCondition).to.be.true;
+
+        componentUnderTest.unmount();
+    })
+  });
+});
+```
+
+- use `describe` to organize tests by application and feature
+  - two levels is adequate depending on the size of the application being tested
+- use `it` to describe the unit test
+  - use active voice
+  - describe the behavior in terms specific to the unit
+  - don't abstract the test description with business logic
+  - **Recommended**:
+    - `it('truncates the address property when it is longer than 15 characters')`
+    - `it('renders an error when props.errors contains at least one item')`
+  - **Not recommended**:
+    - `it('shortens the address when the user has a long address')`
+    - `it('shows an error when the user is not logged in')`
+  - use Enzyme's [shallow](https://airbnb.io/enzyme/docs/api/shallow.html) instead of [mount](https://airbnb.io/enzyme/docs/api/mount.html) when possible to test components
+  - always `unmount` components at the end of the test
+
+## Unit testing forms with schemaform-utils
```

Comment:
Maybe we should title this something like "writing unit tests for VAFS pages"? It's not clear why you would read this section if you're just skimming the headings.